rules/windows/process_creation/proc_creation_win_rundll32_susp_shellexec_execution.yml

```
@@ -11,7 +13,7 @@ references:
     - https://github.com/SigmaHQ/sigma/issues/1009
 author: Nasreddine Bencherchali (Nextron Systems)
 date: 2022-09-01
-modified: 2022-12-30
+modified: 2024-11-25
```

```suggestion
modified: 2022-12-30
```
No need to `modified` for the related field.